Mathlib/CategoryTheory/Galois/Prorepresentability.lean

```diff
@@ -177,6 +188,220 @@ noncomputable def autGaloisSystem : PointedGaloisObject F ⥤ GroupCat.{u₂} wh
     ext (σ : Aut A.obj)
     simp
 
+/-- The limit of `autGaloisSystem` computed in `GroupCat.{max u₁ u₂}`. -/
+noncomputable def autGalois : Type (max u₁ u₂) :=
+  (autGaloisSystem F ⋙ forget _).sections
+
+noncomputable instance : Group (autGalois F) :=
+  inferInstanceAs <| Group (autGaloisSystem F ⋙ forget _).sections
+
+/-- The canonical projection from `autGalois F` to the `C`-automorphism group of each
+pointed Galois object. -/
+noncomputable def autGalois.π (A : PointedGaloisObject F) : autGalois F →* Aut (A : C) :=
+  GroupCat.sectionsπMonoidHom (autGaloisSystem F) A
+
+/- Not a `simp` lemma, because we usually don't want to expose the internals here. -/
+lemma autGalois.π_apply (A : PointedGaloisObject F) (x : autGalois F) :
+    autGalois.π F A x = x.val A :=
+  rfl
+
+lemma autGaloisSystem_map_surjective ⦃A B : PointedGaloisObject F⦄ (f : A ⟶ B) :
+    Function.Surjective ((autGaloisSystem F).map f) := by
+  intro (φ : Aut B.obj)
+  obtain ⟨ψ, hψ⟩ := autMap_surjective_of_isGalois f.val φ
+  use ψ
+  simp only [autGaloisSystem_map]
+  exact hψ
+
+/-- `autGalois.π` is surjective for every pointed Galois object. -/
+theorem autGalois.π_surjective (A : PointedGaloisObject F) :
+    Function.Surjective (autGalois.π F A) := fun (σ : Aut A.obj) ↦ by
+  have (i : PointedGaloisObject F) : Finite ((autGaloisSystem F ⋙ forget _).obj i) :=
+    inferInstanceAs <| Finite (Aut (i.obj))
+  exact eval_section_surjective_of_surjective
+    (autGaloisSystem F ⋙ forget _) (autGaloisSystem_map_surjective F) A σ
+
+/-- Equality of elements of `autGalois F` can be checked on the projections on each pointed
+Galois object. -/
+lemma autGalois_ext {f g : autGalois F}
+    (h : ∀ (A : PointedGaloisObject F), autGalois.π F A f = autGalois.π F A g) : f = g := by
+  dsimp only [autGalois]
+  ext A
+  exact h A
+
+section EndAutGaloisIsomorphism
+
+/-!
+
+### Isomorphism between `Aut F` and `autGalois F`
+
+In this section we establish the isomorphism between the automorphism group of `F` and
+the limit over the automorphism groups of all Galois objects.
+
+We first establish the isomorphism between `End F` and `autGalois F`, from which we deduce that
+`End F` is a group, hence `End F = Aut F`. The isomorphism is built in multiple steps:
+
+- `endEquivLimitFibers : End F ≅ limit (incl F ⋙ F')`: the endomorphisms of
```

Comment:
As we replaced `limit` by `sections`, the docstring should be updated. Otherwise, the PR looks good to me.

Reply:
Updated the docstring and renamed one of the isomorphisms.